go.mod

```diff
@@ -34,7 +34,7 @@ require (
 	github.com/cosmos/ibc-go/v6 v6.2.0
 	github.com/rs/zerolog v1.30.0
 	github.com/tendermint/tendermint v0.34.28
-	golang.org/x/exp v0.0.0-20230515195305-f3d0a9c9a5cc
+	golang.org/x/exp v0.0.0-20230810033253-352e893a4cad
```

Comment:
I think this bump may have caused the test workflow to fail [here](https://github.com/celestiaorg/celestia-app/actions/runs/5854000657/job/15869090226?pr=2259#step:4:2010). So we need to wait for a release with https://github.com/cosmos/gogoproto/pull/83

Ref: 
- https://github.com/ethereum/go-ethereum/pull/27909
- https://github.com/cosmos/gogoproto/blob/28c37ed426b256e2d84b3bb4e39160cb722b7124/proto/merge.go#L260